File: Data Structures & Algorithms/missing-number/submission-1.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = {}
        f = 0
        for i in range(len(nums)+1):
            n[i] = 0
            if i in nums:
                n[i] += 1
        
        for i in n:
            if n[i] == 0:
                f = i
        return f
                

File: Data Structures & Algorithms/rotate-array/submission-1.java
class Solution {
    public void rotate(int[] nums, int k) {
        k = k % nums.length;
        while (k>0) {
            int tmp = nums[nums.length-1];
            int i = nums.length-1;
            while (i > 0) { 
                nums[i] = nums[i-1];
                i--;
            }
            nums[0] = tmp;
            k--;
            
            
        }
        
        //while first el is not nums[k]
        //store last item in tmp
        //if i < nums.length-1
        //shift every value to the right by doing nums[i+1] = nums[i]
        //make first el tmp
        
    }
}